cw-orch/src/osmosis_test_tube/core.rs

```diff
@@ -326,6 +327,64 @@ impl<S: StateInterface> TxHandler for OsmosisTestTube<S> {
     }
 }
 
+impl BankQuerier for OsmosisTestTube {
+    fn balance(
+        &self,
+        address: impl Into<String>,
+        denom: Option<String>,
+    ) -> Result<Vec<cosmwasm_std::Coin>, <Self as TxHandler>::Error> {
+        if let Some(denom) = denom {
+            Ok(vec![Coin {
+                amount: self.query_balance(&address.into(), &denom)?,
+                denom,
+            }])
+        } else {
+            self.query_all_balances(&address.into())
+        }
+    }
+
+    fn supply_of(
+        &self,
+        denom: impl Into<String>,
+    ) -> Result<cosmwasm_std::Coin, <Self as TxHandler>::Error> {
+        let denom: String = denom.into();
+        let supply_of_result: QuerySupplyOfResponse = self
+            .app
+            .borrow()
+            .query(
+                "/cosmos.bank.v1beta1.Query/SupplyOf",
+                &QuerySupplyOfRequest {
+                    denom: denom.clone(),
+                },
+            )
+            .map_err(map_err)?;
+
+        Ok(supply_of_result
+            .amount
+            .map(|c| {
```

Comment:
I remember there being a helper function for this conversion

Reply:
Yeah but it's for cosmrs and not osmosis std :/ Adding such function

Reply:
Added on because it's used above as well